packages/react/src/components/Pagination/Pagination.js

```diff
@@ -127,6 +128,11 @@ export default class Pagination extends Component {
      * `true` if the select box to change the page should be disabled.
      */
     pageInputDisabled: PropTypes.bool,
+
+    /**
+     * Optional prop to set the `size` attribute on the page number select element.
+     */
+    selectSize: PropTypes.number,
```

Comment:
Would it make sense for this prop to be something like `numSelectPagesVisible`? Was hard to get an idea of what this prop did initially (for me) so was trying to brainstorm what might make sense here.

Reply:
Yeah `selectSize` could use some work but on the upside it references both the element and the attribute being affect here.

The thing about `numSelectPagesVisible` is it makes it sound like the prop is applying to a number input? Or moreso, that it involves setting the number of pages visible? 🤔 

Plus I wonder if we shouldn't include the other select ("items per page") in this as well.

What about `setSelectVisibleSize`? Used like this --
```
/** @type {object<number, number>} */
setSelectVisibleSize: PropTypes.shape({
  pageSelect: PropTypes.number,
  itemsPerPageSelect: PropTypes.number,
});
```

Too much? 🤔 Is the name a little better?

Reply:
> Or moreso, that it involves setting the number of pages visible?

As, is that not what the prop is for? Thought it was for setting the max number of pages shown visually to a user at a single time for a select input

Reply:
Basically all I mean is that if we could name it according to the use-case as specific elements can/could change but the use-case still remains, you know?

Reply:
By "Or moreso, that it involves setting the number of pages visible?", I mean that the prop name `numSelectPagesVisible` seems to generally apply to core `Pagination` design / functionality outside of the `size` attribute on the inner select elements. The part "pages visible" in particular is the confusing to me -- I would ask, if I saw that, "but isn't that was the 'items per page' select does? does this prop show multiple pages of content at once?"

And then the question remains  -- how would that prop be expanded to include setting the `size` attribute on the other select element ("items per page"). 

That's why I was asking about `setSelectVisibleSize` with those keys...
The combination of the prop with those keys is more readable for me, it can almost be it's own sentence... 

* Question: "What does the prop do?"
* Answer: "It can **set select visible size** on the **page select** or the **items per page select**"

Reply:
@jendowns right, what I'm suggesting is that currently the component uses `<select>` but this may not necessarily always be the case. Definitely not tied to the name specified above, just a first thought (and even my own suggestion used select accidentally 😂)

I was trying to figure out what the use-case was and made sure the name was tied to that as it would most likely apply to any type of list-box/dropdown we could use here (even though right now it's totally a `<select>`)

An incredibly verbose version of this might be: `numberOfPagesShownForPageSelection`.

Reply:
I like the verbose option `numberOfPagesShownForPageSelection` 😆 
Or maybe something shorter: `numVisibleSelectionOptionsShown` / `numberOfVisibleSelectionOptionsShown`? "Options" is related to a `Select` element but technically is generic enough to apply to something else? 🤔 

(But what about the other select element, "items per page"? What should the prop for *that* `size` attribute be called? 😅 )

Reply:
Honestly I don't have a super strong preference either way. Let me know what you want it (plus the "items per page" select) to be and I can update it! Either way the prop description would explain more about the purpose.

Reply:
I think keeping "size" in the prop name still makes sense even if the underlying element may not be a `<select>` since it would be referring to the functionality of the `size` attribute right? (specifying the number of options shown at once)